Assets/Scripts/UI/ObjectiveList/ObjectiveList.cs

```diff
@@ -21,11 +25,13 @@ void Awake()
 
     void Update()
     {
-        if (((Input.GetAxis(Constants.INPUT_AXIS_HORIZONTAL_DPAD) < -0.75f) || Input.GetKeyDown("j")) && !root.GetBool(Constants.ANIMATION_OBJECTIVELIST_SLIDE))
+        previousAxis = currentAxis; 
+        currentAxis = Input.GetAxis(Constants.INPUT_AXIS_HORIZONTAL_DPAD);
+        if ((currentAxis < - LEFT_DPAD) && (previousAxis >= LEFT_DPAD) && !root.GetBool(Constants.ANIMATION_OBJECTIVELIST_SLIDE))

```

Comment:
```suggestion
        if ((currentAxis < LEFT_DPAD) && (previousAxis >= LEFT_DPAD) && !root.GetBool(Constants.ANIMATION_OBJECTIVELIST_SLIDE))

```

you dont need the - before the first LEFT_DPAD in this if check! doesnt work otherwise actually, just tested!
also same thing below!